src/components/store/OrderCard/index.tsx

```diff
@@ -28,28 +31,38 @@ export const orderStatusColor: { [_ in OrderStatus]: string } = {
 
 interface OrderCardProps {
   order: PublicOrder;
+  focusedOrder?: PublicOrderWithItems;
```

Comment:
hmm, why force at most one order card to be open at a time? [exclusive accordions hurt accessibility and usability](https://github.com/openui/open-ui/issues/925), and it'd also perhaps be easier just for each `OrderCard` to keep track of their own state

Reply:
Done, I'm writing this with `useEffect` and `useState` so we can stagger the number of getOrder API calls to be on demand instead of all at once.